src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const count = 40;
+  const discount = count * days;
+
+  if (days >= 3 && days <= 6) {
+    return discount - 20;
+  }
+
+  if (days >= 7) {
+    return discount - 50;
+  }
+
+  return count * 2;
```

Comment:
The return statement should be 'return discount;' instead of 'return count * 2;' to return the correct cost for days less than 3. The current implementation is incorrect and does not follow the task description.